03/app.js

```diff
@@ -42,10 +42,17 @@ books.getAuthor = function(isbn) {
 }
 
 books.getTitle = function(isbn, lang) {
-
+    const title = this[isbn]['title']['pl'];
+    if (title) {
+        return title;
+    }
```

Comment:
Doby kierunke, ale trzeba pamiętać, że te elementy nie muszą występować tj. być poprawne chodzi o `this[isbn]` czy `this[isbn]['title']`. Pamiętaj też, że mamy zmienną `lang`, która jest dynamiczna (zmienia się) więc należało jej użyć tj. `this[isbn]['title'][lang]`